src/dashboard-refactor/lists-sidebar/components/lists-sidebar-item/index.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { PureComponent } from 'react'
+import styled, { css, keyframes } from 'styled-components'
+
+import colors from '../../../colors'
+import { fonts } from '../../../styleConstants'
+
+import {
+    HoverState,
+    DroppableState,
+    SelectedState,
+    NewItemsCountState,
+} from 'src/dashboard-refactor/types'
+import { MoreActionButtonState } from './types'
+
+import Margin from 'src/dashboard-refactor/components/Margin'
+
+// probably want to use timing function to get this really looking good. This is just quick and dirty
+const blinkingAnimation = keyframes`
+    0% {
+        background-color: ${colors.onHover};
+    }
+    70% {
+        background-color: transparent;
+    }
+    100% {
+        background-color: ${colors.onHover};
+    }
+`
+
+const Container = styled.div`
```

Comment:
I don't know whether this is something in other components, but I'd always put in type information, like `styled.div<{ isHovered: boolean }>`